### PR TITLE
feat(tpp-snap): added `onContentChange` handler

### PR DESCRIPTION
**Pull request by christian-breidler**

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -47,10 +48,13 @@ class App extends TsxComponent<AppProps> {
   @Prop({ required: true }) defaultLocale!: AppProps["defaultLocale"];
   @Prop({ required: true }) handleRouteChange!: AppProps["handleRouteChange"];
   @Prop() fsTppVersion: AppProps["fsTppVersion"];
+  @Prop({ default: false }) customSnapHooks!: AppProps["customSnapHooks"];
```
Comment (konstantin-tkachuk) [inline src/components/App.tsx:51]: You wrote in your comment in JIRA: `this defaults could be opt-out via fsxa.customSnapHooks`
I see this reflected in the code, but I am confused from where and how this flag is meant to be passed to this App component. Specifically, suppose I want to start the fsxa-pwa with `customSnapHooks: true`. Where do I need to set this flag and using which mechanism will it be passed through to the component?

I tried looking at how other props are passed through and currently it seems to me that we need an additional PR in the fsxa-nuxt-module that will add this passthrough logic.

Comment (konstantin-tkachuk) [inline src/components/App.tsx:51]: Additionally, I think we need to add a short documentation regarding these default hooks and how to override them here:  https://github.com/e-Spirit/fsxa-pwa/blob/master/docs/modules/ROOT/pages/advanced/TPP-Snap.adoc

**Review by konstantin-tkachuk — COMMENTED** [review]
Looks good in general. I also tested it manually in the fsxa-pwa and it worked as expected.

File: src/components/App.tsx (hunk 1 above)
Comment (henczi-espirit) [reply src/components/App.tsx:51]: @christian-breidler  will check this

Comment (christian-breidler) [reply src/components/App.tsx:51]: I added this https://github.com/e-Spirit/fsxa-nuxt-module/pull/92.

Comment (christian-breidler) [reply src/components/App.tsx:51]: Please doublecheck https://github.com/e-Spirit/fsxa-pwa/pull/121.

**Pull request merged**